fiftyone/core/annotation.py

```
@@ -153,7 +153,11 @@ def compute_annotation_schema(collection, field_name, scan_samples=True):
             fof.IntField,
         ),
     ):
-        return {"type": "input", "default": None}
+        schema = {"type": "input", "default": None, "ftype": field.__class__.__name__.replace("Field", "").lower()}
```

Pylint (4.0.3)

[convention] Line too long (115/79)

(C0301)